libs/wingc/src/parser.rs

```diff
@@ -985,7 +995,15 @@ impl Parser<'_> {
 		for node in iter {
 			if !self.error_nodes.borrow().contains(&node.id()) {
 				if node.is_error() {
-					_ = self.add_error::<()>(String::from("Unknown parser error."), &node);
+					if node.named_child_count() == 0 {
+						_ = self.add_error::<()>(String::from("Unknown parser error."), &node);
```

Comment:
Why do we need to assign to _?

Curious...

Reply:
`self.add_error` returns a `Result`, rust doesn't like you let Results hang out without using them somehow